packages/portal/src/pages/account/index.vue

```diff
@@ -138,24 +112,18 @@
     name: 'AccountIndexPage',
 
     components: {
-      AlertMessage,
       BNav,
       BNavItem,
       ClientOnly,
-      ItemPreviewInterface,
       UserHeader,
+      UserLikes,
       UserSets
     },
 
     mixins: [
       pageMetaMixin
     ],
 
-    beforeRouteLeave(_to, _from, next) {
-      this.$store.commit('set/setSelected', []);
-      next();
-    },
-
```

Comment:
This should be restored. It resets the selected items (item multi-select feature) on route leave.